resources/definitions/strateo3d_IDEX420.def.json

```
@@ -0,0 +1,132 @@
+            "material_print_temperature_layer_0": { "minimum_value": "41", "maximum_value_warning": "300", "maximum_value": "450" },
+            "material_bed_temperature": { "minimum_value": "0", "maximum_value_warning": "120", "maximum_value": "140" },
+            "material_bed_temperature_layer_0": { "minimum_value": "0", "maximum_value_warning": "120", "maximum_value": "140" },
+            "material_final_print_temperature": { "value": "material_print_temperature - 10" },
+            "material_initial_print_temperature": { "value": "material_print_temperature - 10" },
+            "meshfix_maximum_resolution": { "value": 0.5 },
+            "meshfix_maximum_deviation": { "default_value": 0.04 },
+            "prime_tower_min_volume": { "default_value": 35.6 },
+            "prime_tower_position_x": { "value": "machine_depth / 2 - max(extruderValue(adhesion_extruder_nr, 'brim_width') * extruderValue(adhesion_extruder_nr, 'initial_layer_line_width_factor') / 100 if adhesion_type == 'brim' or (prime_tower_brim_enable and adhesion_type != 'raft') else (extruderValue(adhesion_extruder_nr, 'raft_margin') if adhesion_type == 'raft' else (extruderValue(adhesion_extruder_nr, 'skirt_gap') if adhesion_type == 'skirt' else 0)), max(extruderValues('travel_avoid_distance'))) - max(extruderValues('support_offset')) - sum(extruderValues('skirt_brim_line_width')) * extruderValue(adhesion_extruder_nr, 'initial_layer_line_width_factor') / 100 - (resolveOrValue('draft_shield_dist') if resolveOrValue('draft_shield_enabled') else 0) - max(map(abs, extruderValues('machine_nozzle_offset_x'))) - 1" },
+            "prime_tower_position_y": { "value": "machine_depth / 2 - prime_tower_size - max(extruderValue(adhesion_extruder_nr, 'brim_width') * extruderValue(adhesion_extruder_nr, 'initial_layer_line_width_factor') / 100 if adhesion_type == 'brim' or (prime_tower_brim_enable and adhesion_type != 'raft') else (extruderValue(adhesion_extruder_nr, 'raft_margin') if adhesion_type == 'raft' else (extruderValue(adhesion_extruder_nr, 'skirt_gap') if adhesion_type == 'skirt' else 0)), max(extruderValues('travel_avoid_distance'))) - max(extruderValues('support_offset')) - sum(extruderValues('skirt_brim_line_width')) * extruderValue(adhesion_extruder_nr, 'initial_layer_line_width_factor') / 100 - (resolveOrValue('draft_shield_dist') if resolveOrValue('draft_shield_enabled') else 0) - max(map(abs, extruderValues('machine_nozzle_offset_y'))) - 3" },
+			"retraction_amount": { "default_value": 1.5 },
+            "retraction_combing": { "default_value": "all" },
+            "retraction_combing_max_distance": { "default_value": 5 },
+            "retraction_count_max": { "default_value": 15 },
+            "retraction_hop": { "value": "2" },
+            "retraction_hop_enabled": { "value": "extruders_enabled_count > 1"  },
+            "retraction_hop_only_when_collides": { "value": "True" },
+            "retraction_min_travel": { "value": "3*line_width" },
```

```suggestion
            "retraction_min_travel": { "value": "3 * line_width" },
```
Use consistent spacing for formulas. I've not highlighted all of them, but there are quite a few of them out there.

---

Hello nallath, every formula corrected in :
strateo3d.def.json
strateo3d_IDEX420.def.json
strateo3d_IDEX420_duplicate.def.json
strateo3d_IDEX420_mirror.def.json